config/crd/patches/webhook_in_akodeploymentconfigs.yaml

```diff
@@ -3,15 +3,17 @@
 apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
-  name: akodeploymentconfigs.network.tanzu.vmware.com
+  name: akodeploymentconfigs.networking.tkg.tanzu.vmware.com
 spec:
   conversion:
     strategy: Webhook
```

Comment:
We don't need to do the conversion for ADC so far, since the only ADC version is `v1alpha1`. I recommend to change the strategy to be `None`, like [this example](https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definition-versioning/).